### ISSUE TITLE
angular: An error is observed on the browser console after navigation to Metrics page

### ISSUE THREAD

**Issue by vishal423** [jhipster/generator-jhipster#9146]
##### **Overview of the issue**
Following strack trace is seen after navigation to Metrics page:

```
ERROR 
TypeError: can't convert undefined to object
Stack trace:
JhiKeysPipe.prototype.transform@webpack-internal:///./node_modules/ng-jhipster/fesm5/ng-jhipster.js:1330:25
checkAndUpdatePureExpressionInline@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:23047:33
checkAndUpdateNodeInline@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:23619:20
checkAndUpdateNode@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:23577:16
debugCheckAndUpdateNode@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:24211:19
debugCheckDirectivesFn@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:24171:13
View_JhiMetricsEndpointsRequestsComponent_1/<@ng:///NgJhipsterModule/JhiMetricsEndpointsRequestsComponent.ngfactory.js:70:49
debugUpdateDirectives@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:24163:12
checkAndUpdateView@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:23559:5
callViewAction@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:23800:21
execEmbeddedViewsAction@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:23763:17
checkAndUpdateView@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:23560:5
callViewAction@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:23800:21
execComponentViewsAction@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:23742:13
checkAndUpdateView@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:23565:5
callViewAction@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:23800:21
execComponentViewsAction@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:23742:13
checkAndUpdateView@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:23565:5
callViewAction@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:23800:21
execEmbeddedViewsAction@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:23763:17
checkAndUpdateView@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:23560:5
callViewAction@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:23800:21
execComponentViewsAction@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:23742:13
checkAndUpdateView@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:23565:5
callWithDebugContext@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:24429:22
debugCheckAndUpdateView@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:24131:12
ViewRef_.prototype.detectChanges@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:21940:13
ApplicationRef.prototype.tick/<@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:18371:58
ApplicationRef.prototype.tick@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:18371:13
next/<@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:18262:99
ZoneDelegate.prototype.invoke@webpack-internal:///./node_modules/zone.js/dist/zone.js:390:17
onInvoke@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:17639:24
ZoneDelegate.prototype.invoke@webpack-internal:///./node_modules/zone.js/dist/zone.js:389:37
Zone.prototype.run@webpack-internal:///./node_modules/zone.js/dist/zone.js:149:24
NgZone.prototype.run@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:17553:16
next@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:18262:69
EventEmitter.prototype.subscribe/schedulerFn<@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:13854:36
SafeSubscriber.prototype.__tryOrUnsub@webpack-internal:///./node_modules/rxjs/_esm5/internal/Subscriber.js:206:13
SafeSubscriber.prototype.next@webpack-internal:///./node_modules/rxjs/_esm5/internal/Subscriber.js:144:17
Subscriber.prototype._next@webpack-internal:///./node_modules/rxjs/_esm5/internal/Subscriber.js:87:9
Subscriber.prototype.next@webpack-internal:///./node_modules/rxjs/_esm5/internal/Subscriber.js:64:13
Subject.prototype.next@webpack-internal:///./node_modules/rxjs/_esm5/internal/Subject.js:58:17
EventEmitter.prototype.emit@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:13838:54
checkStable@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:17608:13
onLeave@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:17675:5
onInvokeTask@webpack-internal:///./node_modules/@angular/core/fesm5/core.js:17633:17
ZoneDelegate.prototype.invokeTask@webpack-internal:///./node_modules/zone.js/dist/zone.js:421:41
Zone.prototype.runTask@webpack-internal:///./node_modules/zone.js/dist/zone.js:194:28
ZoneTask.invokeTask@webpack-internal:///./node_modules/zone.js/dist/zone.js:497:24
invokeTask@webpack-internal:///./node_modules/zone.js/dist/zone.js:1743:9
globalZoneAwareCallback@webpack-internal:///./node_modules/zone.js/dist/zone.js:1780:21
JhiMetricsEndpointsRequestsComponent.html:13:12
View_JhiMetricsEndpointsRequestsComponent_1
JhiMetricsEndpointsRequestsComponent.html:13:12
DebugContext_.prototype.logError
core.js:24041
ErrorHandler.prototype.handleError
core.js:15762
ApplicationRef.prototype.tick/<
core.js:18028:54
ZoneDelegate.prototype.invoke
zone.js:391
Zone.prototype.run
zone.js:150
NgZone.prototype.runOutsideAngular
core.js:17248
ApplicationRef.prototype.tick
core.js:18028
next/<
core.js:17912:97
ZoneDelegate.prototype.invoke
zone.js:391
onInvoke
core.js:17289
ZoneDelegate.prototype.invoke
zone.js:390
Zone.prototype.run
zone.js:150
NgZone.prototype.run
core.js:17203
next
core.js:17912:67
EventEmitter.prototype.subscribe/schedulerFn<
core.js:13504:34
SafeSubscriber.prototype.__tryOrUnsub
Subscriber.js:196
SafeSubscriber.prototype.next
Subscriber.js:134
Subscriber.prototype._next
Subscriber.js:77
Subscriber.prototype.next
Subscriber.js:54
Subject.prototype.next
Subject.js:47
EventEmitter.prototype.emit
core.js:13488:52
checkStable
core.js:17258
onLeave
core.js:17325
onInvokeTask
core.js:17283
ZoneDelegate.prototype.invokeTask
zone.js:422
Zone.prototype.runTask
zone.js:195
ZoneTask.invokeTask
zone.js:498
invokeTask
zone.js:1744
globalZoneAwareCallback
zone.js:1781
ERROR CONTEXT 
Object { view: {…}, nodeIndex: 25, nodeDef: {…}, elDef: {…}, elView: {…} }
JhiMetricsEndpointsRequestsComponent.html:13:12
```

##### **Reproduce the error**
Login with admin credentials and navigate to Administration --> Metrics 

##### **JHipster Version(s)**
5.8.0

##### **JHipster configuration**

```
{
    "generator-jhipster": {
        "promptValues": {
            "packageName": "com.mycompany.myapp",
            "nativeLanguage": "en"
        },
        "jhipsterVersion": "5.8.0",
        "applicationType": "monolith",
        "baseName": "hip_test",
        "packageName": "com.mycompany.myapp",
        "packageFolder": "com/mycompany/myapp",
        "serverPort": "8080",
        "authenticationType": "jwt",
        "cacheProvider": "ehcache",
        "enableHibernateCache": true,
        "websocket": false,
        "databaseType": "sql",
        "devDatabaseType": "h2Disk",
        "prodDatabaseType": "postgresql",
        "searchEngine": false,
        "messageBroker": false,
        "serviceDiscoveryType": false,
        "buildTool": "maven",
        "enableSwaggerCodegen": false,
        "jwtSecretKey": "dummy",
        "clientFramework": "angularX",
        "useSass": true,
        "clientPackageManager": "yarn",
        "testFrameworks": ["protractor"],
        "jhiPrefix": "jhi",
        "otherModules": [],
        "enableTranslation": true,
        "nativeLanguage": "en",
        "languages": ["en", "hi"],
        "entitySuffix": "",
        "dtoSuffix": "DTO"
    }
}
```

##### **Browsers and Operating System**
- Firefox 60 ESR

-   [x] Checking this box is mandatory (this is just to show you read everything)

**Comment by clement26695** [comment]
Thank you for reporting the issue. It should be fixed with #9130

**Comment by vishal423** [comment]
@clement26695 , yeah, it's same issue
